vite.config.js

```diff
@@ -10,6 +10,9 @@ export default defineConfig({
       name: "network-gas-price",
       fileName: "network-gas-price",
     },
+    rollupOptions: {
+      external: ["isomorphic-unfetch"],
+    },
```

Comment:
Awesome 👌🏻 

However we also need for a wider compatibiliy:
 
```suggestion
    rollupOptions: {
      external: ["isomorphic-unfetch"],
      output: {
        globals: {
         "isomorphic-unfetch": "IsomorphicUnfetch",
        },
      },
    },
```

Reply:
Actually, we are going to build it for now. It's not clear to me the best way to add it as an external dependency at the moment.